Source/Shared/arcana/threading/cancellation.h

```diff
@@ -56,7 +56,7 @@ namespace arcana
 
     protected:
         cancellation() = default;
-        cancellation& operator=(const cancellation&) = default;
+        cancellation& operator=(const cancellation&) = delete;
```

Comment:
I guess `cancellation` is not copyable? Was this just a pre-existing bug?

Reply:
Yes. The clang compiler throws a warning because std::mutex is not copyable. Using `= default` just makes it not copyable also.